demosplan/DemosPlanCoreBundle/ResourceTypes/StatementResourceType.php

```diff
@@ -271,12 +281,47 @@ protected function getProperties(): array|ResourceConfigBuilderInterface
             $configBuilder->originalId
                 ->readable(true)->aliasedPath(Paths::statement()->original->id);
             $configBuilder->paragraphParentId
-                ->readable(true)->aliasedPath(Paths::statement()->paragraph->paragraph->id);
+                ->readable(true)->aliasedPath(Paths::statement()->paragraph->paragraph->id)
+                ->updatable([$simpleStatementCondition], function (Statement $statement, ?string $paragraphParentId): array {
+                    if (null === $paragraphParentId || '' === $paragraphParentId) {
+                        $statement->setParagraph(null);
+                    } else {
+                        $paragraphEntity = $this->paragraphRepository->get($paragraphParentId);
+                        Assert::notNull($paragraphEntity);
+                        $paragraphVersion = $this->paragraphVersionRepository->createVersion($paragraphEntity);
+                        $statement->setParagraph($paragraphVersion);
+                    }
+
+                    return [];
+                });
             $configBuilder->paragraphTitle
                 ->readable(true)->aliasedPath(Paths::statement()->paragraph->title);
             $configBuilder->assignee->readable()->filterable();
             $configBuilder->authorName->readable(true)->filterable();
             $configBuilder->submitName->readable(true)->filterable()->sortable();
+            $configBuilder->location
+                ->readable(true, fn (Statement $statement): ?array => $this->coordinateJsonConverter->convertJsonToCoordinates($statement->getPolygon()))
+                ->aliasedPath(Paths::statement()->polygon);
+
+            if ($this->currentUser->hasPermission('field_procedure_elements')) {
+                // @todo double check permission to update
```

Comment:
Is this todo finished?

Reply:
@breierDemos I double checked and decided to remove the permission check that I introduced in this PR. Tihs permission field_procedure_elements is too general that does not need login. $configBuilder->elements  is already under $this->hasAssessmentPermission() and that is restrictive enough ass111d01ea72ae19d4b786aa99a0e9e83094c1a10f